pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -189,9 +195,7 @@ func initFSS(ctx context.Context, k8sClient clientset.Interface, controllerClust
 
 	// Set up kubernetes resource listeners for k8s orchestrator
 	k8sOrchestratorInstance.informerManager = k8s.NewInformer(k8sClient)
-	// TODO: Restrict the Configmap listener to the CSI namespace.
```

Comment:
As a part of this change, can you also convert all the `ClusterRole` and `ClusterRoleBinding` resources to `Role` and `RoleBinding` resources in manifest YAMLs wherever necessary?

Reply:
For example, in vsphere 7.0U2, vanilla and GC node YAMLs do not need `ClusterRole` anymore.

Reply:
> As a part of this change, can you also convert all the ClusterRole and ClusterRoleBinding resources to Role and RoleBinding resources in manifest YAMLs wherever necessary?

How is this related to the changes in the PR ?

Reply:
As the informer we previously had was watching for configmaps in all namespaces, we had to use a `ClusterRole` in admission webhook rbac and in node rbac rules. Now that you have restricted the informer to the CSI pod namespace, you can change them to `Role` and `RoleBinding` for that namespace. 
Refer to https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/362#discussion_r508109059 for more details